modules/ppcp-googlepay/resources/js/GooglepayButton.js

```diff
@@ -218,15 +212,15 @@ class GooglepayButton {
     async onButtonClick() {
```

Comment:
Maybe remove also this `async` as it seems not using `await` inside.

Reply:
Good catch!